ext/date/php_date.c

```diff
@@ -2545,6 +2547,27 @@ PHP_METHOD(DateTime, createFromImmutable)
 }
 /* }}} */
 
+/* {{{ proto DateTime::createFromInterface(DateTimeInterface object)
+   Creates new DateTime object from an existing DateTimeInterface object.
+*/
+PHP_METHOD(DateTime, createFromInterface)
+{
+    zval *datetimeinterface_object = NULL;
+    php_date_obj *new_obj = NULL;
+    php_date_obj *old_obj = NULL;
+
+    ZEND_PARSE_PARAMETERS_START(1, 1)
+            Z_PARAM_OBJECT_OF_CLASS(datetimeinterface_object, date_ce_interface)
```

Comment:
This looks like it's indented with space, not tabs. Please use tabs.

Reply:
I fixed the function I added but I noticed that others where indented with space rather than tabs. Should I fix those too ?